### DISCUSSION TITLE
Intellisense window is compressed into an unreadable state in Vs code

### DISCUSSION THREAD

**Discussion by Ngozistephen** [microsoft/vscode-discussions#783] [Extension Development QnA] [answered]
[](url)
![Screenshot 2023-08-28 041728](https://github.com/microsoft/vscode-discussions/assets/60478145/91c7b7ec-8082-4812-b750-92b60d6a4ccd)

I have been trying to solve it

**Comment by gjsjohnmurray** [comment]
Are you developing an extension?

**Reply by Ngozistephen** [reply]
No, just building a project and I noticed that my suggested words are compressed I can't see anything

**Comment by gjsjohnmurray** [comment, marked as answer]
Please open your issue at https://github.com/microsoft/vscode/issues as this forum here is intended for extension developers.

**Comment by olguzzar** [comment]
Closing as this question is off-topic for this forum, which specifically focuses on extension development.
